ffi/blitbuffer.lua

```diff
@@ -658,6 +647,18 @@ end
 function BB_mt.__index:getType()
     return rshift(band(MASK_TYPE, self.config), SHIFT_TYPE)
 end
+
+-- Determine if a pair of buffers can use CBB in relation to each other, or whether CBB is used at all.
+-- Used to skip unsupported modes such as unrelated inverses.
+-- TODO: Possibly some RGB24/32 stuff too?
+function BB_mt.__index:canUseCbbTogether(other)
+    return use_cblitbuffer and self:getInverse() == other:getInverse()
+end
+
+function BB_mt.__index:canUseCbb()
```

Comment:
Not 100% convinced of `Cbb` but that aside. I'll leave any further potential nitting to @pazos and/or @NiLuJe.

Reply:
I utterly failed to find anything better at the time, so, I'm fine with it, FWIW ;).

Reply:
Talking about canUseCBB vs canUseCbb. I prefer the former but I feel I've met my nitpick quota, plus those were more substantial nits. :-P

Reply:
Makes the C stands out more this way, which isn't necessarily a bad thing, I think ;).